src/App.jsx

```diff
@@ -14,70 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const EMPTY_SPACE = '';
+const ADD_BUTTON_TEXT = '+';
+const REMOVE_BUTTON_TEXT = '-';
+const DEFAULT_GOOD_VALUE = 'Jam';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [goodState, setGoodState] = useState(DEFAULT_GOOD_VALUE);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const selectedGoodMessage = `${goodState} is selected`;
+  const noGoodSelectedMessage = 'No goods selected';
```

Comment:
This can also be expressed as a constant outside the component (and written in uppercase).